lib/spack/spack/test/conftest.py

```diff
@@ -938,3 +938,53 @@ def invalid_spec(request):
     """Specs that do not parse cleanly due to invalid formatting.
     """
     return request.param
+
+
+@pytest.fixture("module")
+def mock_test_repo(tmpdir_factory):
```

Comment:
(minor - no changes required) this appears to have been moved out of the `cmd.create` test module although that appears to still be the only place it is still used. If that is true, generally it's best to keep fixtures in the module where they are needed.

Reply:
Oversight.  IIRC I intended to have tests relying on it in more than one module.